bundles/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/handler/GardenaAccountHandler.java

```diff
@@ -51,80 +60,168 @@
 
     private @Nullable GardenaSmart gardenaSmart;
     private HttpClientFactory httpClientFactory;
     private WebSocketFactory webSocketFactory;
 
+    private final Object reInitializationCodeLock = new Object();
+    private @Nullable ScheduledFuture<?> reInitializationTask;
+    private boolean reInitializationCausedBy429 = false;
+    private Instant lastApiCallTime = Instant.MIN;
+    private boolean lastApiCallTimeLoaded = false;
+
     public GardenaAccountHandler(Bridge bridge, HttpClientFactory httpClientFactory,
             WebSocketFactory webSocketFactory) {
         super(bridge);
         this.httpClientFactory = httpClientFactory;
         this.webSocketFactory = webSocketFactory;
     }
 
+    private Instant lastApiCallTime() {
+        if (!lastApiCallTimeLoaded) {
+            Map<String, String> properties = getThing().getProperties();
+            String property = properties.getOrDefault(GardenaBindingConstants.LAST_API_CALL_TIME, "");
+            lastApiCallTime = "".equals(property) ? Instant.now().minus(Duration.ofHours(1)) : Instant.parse(property);
+            lastApiCallTimeLoaded = true;
+        }
+        return lastApiCallTime;
+    }
+
+    private void lastApiCallTimeUpdate() {
+        lastApiCallTime = Instant.now();
+        getThing().setProperty(GardenaBindingConstants.LAST_API_CALL_TIME,
+                lastApiCallTime.truncatedTo(ChronoUnit.SECONDS).toString());
+    }
+
     @Override
     public void initialize() {
         logger.debug("Initializing Gardena account '{}'", getThing().getUID().getId());
-        initializeGardena();
+        Instant now = Instant.now();
+        Instant notBeforeTime = lastApiCallTime().plus(REINITIALIZE_DELAY_MINUTES_BACK_OFF)
+                .plus(REINITIALIZE_DELAY_MINUTES_BACK_OFF);
+        if (now.isBefore(notBeforeTime)) {
+            // delay the initialisation
+            Duration delay = Duration.between(now, notBeforeTime);
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, uiText(delay.getSeconds()));
+            scheduleReinitialize(delay);
+        } else {
+            // do immediate initialisation
+            scheduler.submit(() -> initializeGardena());
+        }
     }
 
     public void setDiscoveryService(GardenaDeviceDiscoveryService discoveryService) {
         this.discoveryService = discoveryService;
     }
 
+    /**
+     * Format a description text to display on the thing main UI page
+     *
+     * @param delaySeconds the delay that will be added to the current time
+     * @return the description text
+     */
+    public String uiText(long delaySeconds) {
+        return "Waiting to make automatic reconnection attempt at " + LocalDateTime.now().plusSeconds(delaySeconds)
+                .truncatedTo(ChronoUnit.SECONDS).format(DateTimeFormatter.ofPattern("HH:MM:SS (cccc)"));
+    }
+
     /**
      * Initializes the GardenaSmart account.
+     * This method is called on a thread.
```

Comment:
Indeed. All methods are ;-)

(Seriously: this comment seems a bit useless?)

Reply:
Yes. 'background'